frontend/src/stock/DeleteItemModal.tsx

```diff
@@ -30,6 +30,8 @@ function DeleteItemModal(props: ModalProps) {
         <Modal
             isOpen={props.modalIsOpen}
             contentLabel="Delete Modal"
+            ariaHideApp={false}
+            onRequestClose={props.closeModal}
         >
```

Comment:
You have two DeleteItemModals, rename them or combine in one. Sonarcloud says there is duplicated code because of two "DeleteItemModals"

Reply:
On my todolist to combine them